.github/workflows/update_uplink_schema.yml

```diff
@@ -0,0 +1,32 @@
+name: Update Uplink Schema
+run-name: ${{ github.actor }} is updating the Uplink schema 🚀
+on:
+  schedule:
+    # On the 5 of every hour
+    - cron:  '5 * * * *'
+jobs:
+  Update-Uplink-Schema:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rover
+        run: |
+          curl -sSL https://rover.apollo.dev/nix/v0.14.1 | sh
+          echo "/home/runner/.rover/bin" >> $GITHUB_PATH
+      - name: Make changes to pull request
+        run: |
+          rover graph introspect https://uplink.api.apollographql.com/ | perl -pe 'chomp if eof' > ./apollo-router/src/uplink/uplink.graphql
```

Comment:
Right now I'm chomping off the trailing newline to gain file equality because the script/test that we currently have is _not_ including the trailing newline while `rover graph introspect` does capture it (not emitting that line would mess up the prompt output on `rover`).

Reply:
I’ll need to trust you that the Perl script does what it looks like it does, but ok

Reply:
@SimonSapin If I actually pull in the removal to the test that asserts the uplink graphql hasn't changed then I can remove that `perl` `chomp` 🦷